Godeps/_workspace/src/k8s.io/kubernetes/pkg/securitycontextconstraints/util/util_test.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"reflect"
+	"testing"
+
+	"k8s.io/kubernetes/pkg/api"
+)
+
+func TestVolumeSourceFSTypeDrift(t *testing.T) {
```

Comment:
I think a comment would be beneficial here, this test is really doing two things:

1.  testing the integrity of `GetAllFSTypesAsSet`
2.  testing that for all the fields of volume source, an fstype pops out